.github/workflows-source/release-testing.yml

```diff
@@ -6,9 +6,6 @@ on:
       - 'hotfix-*-rc--*'
       - 'rc--*'
   workflow_dispatch:
-  pull_request:
-    paths:
-      - '.github/workflows/release-testing.yml'
```

Comment:
We don't want system nightly tests to trigger every time there is an image update.

Reply:
The same goes for schedule-weekly. User can trigger the workflow manually via UI or gh CLI.